packages/react-router-dom/index.tsx

```diff
@@ -27,7 +32,6 @@ import type {
   HashHistory,
   History,
   HydrationState,
-  RouteObject,
```

Comment:
`react-router-dom` now re-exports from `react-router` instead of `@remix-run-/router`